.release-version

```diff
@@ -1 +1 @@
-1.11.0
+1.12.2
```

Comment:
Shouldn't this be 1.12.0?

Reply:
@JamesGlover I was bumping it to trigger the manual build of a release to deploy and test, but I now know to use -rcx, so its a shame its had to bump two minors. Will remember release candidates here on out!